pkgs/tools/networking/scout/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
```

Comment:
Please move the package to `pkgs/by-name`.

Reply:
Done. Still haven't got used to it. Eventually I will...